test/jasmine/tests/pie_test.js

```diff
@@ -1682,26 +1696,31 @@ describe('Test event data of interactions on a pie plot:', function() {
         it('should not emit a hover if you\'re dragging', function() {
             gd._dragging = true;
             mouseEvent('mouseover', pointPos[0], pointPos[1]);
-            expect(futureData).toBeUndefined();
+            expect(futureData).toBe(null);
         });
 
         it('should not emit a hover if hover is disabled', function() {
-            Plotly.relayout(gd, 'hovermode', false);
-            mouseEvent('mouseover', pointPos[0], pointPos[1]);
-            expect(futureData).toBeUndefined();
+            Plotly.relayout(gd, 'hovermode', false)
+            .then(function() {
+                mouseEvent('mouseover', pointPos[0], pointPos[1]);
+                expect(futureData).toBe(null);
+            });
```

Comment:
`done`/`failTest`

Reply:
Fixed in https://github.com/plotly/plotly.js/pull/5340/commits/33b72e1eac53fb93c7b6d71045210e8847312d04